services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -386,6 +387,7 @@ export const generateDynamicUiWorkflow = async (prismaClient: PrismaClient, proj
     data: kybDynamicExample,
   });
 
+  //@ts-ignore
   await generateDynamicUiTest(prismaClient, workflow.id, projectId || workflow.projectId);
```

Comment:
Note if you're up to date with dev the project ID from DB should never be null and only one of these project ids need to be passed..